iib/workers/tasks/opm_operations.py

```diff
@@ -389,3 +389,57 @@ def opm_registry_add_fbc(
         binary_image=binary_image,
         dockerfile_name='index.Dockerfile',
     )
+
+
+def _opm_registry_rm(index_db_path, operators, base_dir):
+    from iib.workers.tasks.utils import run_cmd
+
+    """
+    Generates and runs the opm command to remove operator package from index db provided.
+
+    :param str index_db_path: path where the input index image is temporarily copied
+    :param list operators: list of operator packages to be removed
+    :param base_dir: the base directory to generate the database and index.Dockerfile in.
+    """
+
+    cmd = [
+        'opm',
+        'registry',
+        'rm',
+        '--database',
+        index_db_path,
+        '--packages',
+        ','.join(operators),
+    ]
+    run_cmd(cmd, {'cwd': base_dir}, exc_msg='Failed to add the bundles to the index image')
```

Comment:
`Failed to add` :) but we are removing :)

Reply:
copy paste error :P